### ISSUE TITLE
Add Accuracy metric

### ISSUE THREAD

**Issue by jshin1394** [google/metrax#37]

**Comment by jshin1394** [comment]
Let's start by investigating this [Keras implementation of Accuracy](https://www.tensorflow.org/api_docs/python/tf/keras/metrics/Accuracy).

**Comment by jeffcarp** [comment]
Since CLU already has an Accuracy class would it work it alias that? 
https://github.com/google/CommonLoopUtils/blob/a8152eb825dc04df8acdbb2b42fc89f9aafa37ba/clu/metrics.py#L875

**Comment by jshin1394** [comment]
Yes! I think it will be nice to provide a metrax.Accuracy class so that our users won't have to call CLU directly. Also, this accuracy class will become more distinct from the CLU.Accuracy class once the Metrax Average class is merged after this [PR](https://github.com/google/metrax/pull/39/files).

**Comment by jshin1394** [comment]
The new `Accuracy` class will inherit from the [`base.Average` class](https://github.com/google/metrax/blob/main/src/metrax/base.py#L29)

**Comment by jshin1394** [comment]
added
